.github/workflows/linux_build_test.yml

```
@@ -49,7 +49,7 @@ jobs:
       - name: Checkout repository
```

I think this may fix the linux build test failure

```suggestion

      - name: Add exception for dubious ownership
        run: git config --global --add safe.directory /__w/DAGMC/DAGMC
        
      - name: Checkout repository
```

---

Thanks, but I think this PR has been replaced by another ( #876 ), so will be close by @pshriwise soon.

---

Correct! Closing now.